ClinicalTrial_SE_curation.Rmd

```diff
@@ -318,3 +318,17 @@ qsave(GSE25066_SE, "GSE25066_SE.qs")
 qsave(GSE15622_SE, "GSE15622_SE.qs")
 
 ```
+
+```{r}
+#===== Automatically upload SE objects to Zenodo =======
+remotes::install_github("bhklab/AnnotationGx@development")
```

Comment:
Suggestions:

To prevent reinstalling the package every time this script is run, maybe we want to use `require` like this:
```
if (!require(AnnotationGx)) remotes::install_github("bhklab/AnnotationGx@development")
```

Reply:
Done